x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -638,11 +638,7 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
   ]);
 
   return (
-    <EuiContextMenuPanel
-      hasFocus={discoverUrlError !== undefined || !!openInDiscoverUrl}
-      items={contextMenuItems}
-      data-test-subj="mlAnomaliesListRowActionsMenu"
-    />
+    <EuiContextMenuPanel items={contextMenuItems} data-test-subj="mlAnomaliesListRowActionsMenu" />
```

Comment:
This component is going to need some more refactoring as there is a 'View in Discover' item in the context menu which is built by an `async` call. Currently the item stays in the loading state until one of the other items in the menu is given focus. 

![image](https://user-images.githubusercontent.com/7405507/165278647-82492a8f-a26b-42da-b435-2a074cad76b7.png)

@qn895 could you take a look please and see what changes are required inside the `useEffect` hook to update the loading state of the 'View in Discover' menu item.

Reply:
@constancecchen Can you take a look at these focus management issues?

Reply:
> Could the initialFocus property be added to those two popovers as part of this PR?

I can absolutely take a look, but as a rule we would strongly prefer creating a follow-up issue for this and addressing in a separate PR so as to reduce CI churn/impact on other teams. We will definitely have this fixed before 8.3 FF and as quickly as we can however. Does that sound alright @peteharverson @qn895?

Reply:
Following up from our convo in Slack, tested and can confirm these issues are now resolved with https://github.com/elastic/kibana/pull/130593/commits/df16c89136c01c23ff7baffa9456dc4fabfcf3b2 and https://github.com/elastic/kibana/pull/130593/commits/c9b033172bc661b24d0de14ce37caf5d0bfbddce. Thanks @constancecchen!!!

Reply:
@qn895 @constancecchen another edit is needed to `links_menu.tsx` here - for the case where the 'View in Discover' item stays disabled because no data view exists for the job. I think this can be done by adding `href` to the list of watched items:

```
    <EuiContextMenuPanel
      items={contextMenuItems}
      watchedItemProps={['disabled', 'href']} // Ensures the 'View in Discover' link rerenders when loading completes
      data-test-subj="mlAnomaliesListRowActionsMenu"
    />
```

Reply:
@peteharverson This should be now be fixed here (https://github.com/elastic/kibana/pull/130593/commits/35bd584c59df3ca5bee1a7d3432ff0075f54d769)